website/docs/r/elasticsearch_domain.html.markdown

```diff
@@ -120,6 +120,14 @@ Security Groups and Subnets referenced in these attributes must all be within th
 * `cloudwatch_log_group_arn` - (Required) ARN of the Cloudwatch log group to which log needs to be published.
 * `enabled` - (Optional, Default: true) Specifies whether given log publishing option is enabled or not.
 
+**cognito_options** supports the following attribute:
+
+AWS documentation: [Amazon Cognito Authentication for Kibana](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-cognito-auth.html)
+
```

Comment:
This is missing the `enabled` attribute documentation 😄

Reply:
:man_facepalming: